core/templates/dev/head/pages/topics_and_skills_dashboard/new_topic_name_editor_directive.html

```diff
@@ -3,21 +3,23 @@ <h3>
     Enter Topic Name
   </h3>
 </div>
-<div class="modal-body new-topic-name-editor" ng-class="{'has-error': isTopicNameEmpty(topicName)}">
-  <p>
-    Enter the name for the topic: &nbsp;
-    <input class="form-group" ng-model="topicName" maxlength="20">
-    <p style="color: red;" ng-if="topicName.length === 20">
-      Topic name should be less than 20 characters.
+<form ng-submit="save(topicName)">
+  <div class="modal-body new-topic-name-editor" ng-class="{'has-error': isTopicNameEmpty(topicName)}">
+    <p>
+      Enter the name for the topic: &nbsp;
+      <input class="form-group" ng-model="topicName" maxlength="20" autofocus>
+      <p style="color: red;" ng-if="topicName.length === 20">
+        Topic name should be less than 20 characters.
+      </p>
     </p>
-  </p>
-</div>
-<div class="modal-footer">
-  <button class="btn btn-default" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-success" ng-disabled="isTopicNameEmpty(topicName)" ng-click="save(topicName)">
-    <span>Create Topic</span>
-  </button>
-</div>
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-success" type="submit" ng-disabled="isTopicNameEmpty(topicName)">
```

Comment:
Should this be disabled also if the topic name is more than 20 characters?

Reply:
It can't be more than 20 characters, as I set `maxlength=20` in the input field above, so it just stops taking input when 20 characters is reached.
Though, I will put this check also.

Reply:
Done.